lib/linguist/languages.yml

```
@@ -1650,6 +1650,19 @@ Fortran Free Form:
   codemirror_mode: fortran
   codemirror_mime_type: text/x-fortran
   language_id: 761352333
+FreeBasic:
+  type: programming
+  color: "#867db1"
+  extensions:
+  - ".bi"
+  - ".bas"
+  tm_scope: source.vbnet
+  aliases:
+  - fb
+  ace_mode: text
+  codemirror_mode: vb
+  codemirror_mime_type: text/x-vb
+  language_id: 761352334
```

```suggestion
  language_id: 472896659
```

Just spotted that you set this manually and not using the `script/update-ids` script as detailed in the CONTRIBUTING.md file